Resources/Locale/en-US/job/job-supervisors.ftl

```diff
@@ -10,6 +10,7 @@ job-supervisors-service = chefs, botanists, the bartender, and the head of perso
 job-supervisors-engineering = station engineers, atmospheric technicians, and the chief engineer
 job-supervisors-medicine = medical doctors, chemists, and the chief medical officer
 job-supervisors-security = security officers, the warden, and the head of security
+job-supervisors-security-hire = the head of security and whoever hires you
```

Comment:
i would really just cut out the "whoever hires you" since they aren't your supervisors so much as just clientele.